provider/pkg/kinds/kinds.go

```diff
@@ -103,6 +103,7 @@ const (
 	PriorityClassList                  Kind = "PriorityClassList"
 	PriorityLevelConfiguration         Kind = "PriorityLevelConfiguration"
 	PriorityLevelConfigurationList     Kind = "PriorityLevelConfigurationList"
+	Release                            Kind = "Release"
```

Comment:
I didn't expect this file to require changes. Was something breaking without it?

Reply:
This was a side-effect of codegen changes. Excluded release from being added to kinds or groups.